--- cryptography-2.5.orig/tests/hazmat/backends/test_openssl.py	2019-01-22 14:41:49.000000000 -0200
+++ cryptography-2.5/tests/hazmat/backends/test_openssl.py	2019-01-29 11:24:23.575868536 -0200
@@ -271,12 +271,12 @@
                         'getrandom']
         if sys.platform.startswith('linux'):
             assert name in ['getrandom', '/dev/urandom']
-        if sys.platform == 'darwin':
-            # macOS 10.12+ supports getentropy
-            if parse_version(os.uname()[2]) >= parse_version("16.0"):
-                assert name == 'getentropy'
-            else:
-                assert name == '/dev/urandom'
+        # if sys.platform == 'darwin':
+        #     # macOS 10.12+ supports getentropy
+        #     if parse_version(os.uname()[2]) >= parse_version("16.0"):
+        #         assert name == 'getentropy'
+        #     else:
+        #         assert name == '/dev/urandom'
         if sys.platform == 'win32':
             assert name == 'CryptGenRandom'
 
