From c0aac7fbe675d3afcf5c64eed39f369d706f4fa1 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Thu, 4 Nov 2021 08:33:08 +0100
Subject: [PATCH] Use pyo3 0.15.0 to fix cross-compilation

---
 src/rust/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/rust/Cargo.toml b/src/rust/Cargo.toml
index 6d59072..29c7e29 100644
--- a/src/rust/Cargo.toml
+++ b/src/rust/Cargo.toml
@@ -7,7 +7,7 @@ publish = false
 
 [dependencies]
 lazy_static = "1"
-pyo3 = { version = "0.14.5" }
+pyo3 = { version = "0.15.0" }
 asn1 = { version = "0.6", default-features = false, features = ["derive"] }
 pem = "0.8"
 chrono = { version = "0.4", default-features = false, features = ["alloc"] }
-- 
2.30.1 (Apple Git-130)

